packages/portal-proto/next.config.js

```diff
@@ -4,6 +4,18 @@
  * means that the application will be available at "https://<host>/v2"
  */
 const basePath = process.env.NEXT_PUBLIC_BASEPATH;
+const PROTEINPAINT_API =
+  process.env.PROTEINPAINT_API || process.env.NEXT_PUBLIC_PROTEINPAINT_API;
+const connectSrc = [
+  "https://portal.gdc.cancer.gov",
+  "https://browser-intake-datadoghq.com",
+  "https://api.gdc.cancer.gov",
+  "https://www.google-analytics.com",
+];
+const PROTEINPAINT_HOST = PROTEINPAINT_API.split("://")[1]?.split("/")[0] || "";
+// in SJ dev environment, this would point to a local PP server instance
+if (!connectSrc.includes(`https://${PROTEINPAINT_HOST}`))
+  connectSrc.push(`https://${PROTEINPAINT_HOST}`);
```

Comment:
@siosonel I realize this PR already got merged.  Could you submit another PR that limits the PP parts of this code to execute only when `NODE_ENV === 'development'`?  That will prevent unintentionally affecting production.